Makefile

```diff
@@ -128,3 +128,7 @@ bundle: manifests
 .PHONY: bundle-build
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+cross-build-image:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a -o manager.exe main.go
```

Comment:
Can we rename this to `manager` or `manager.linux` and add to `.gitignore`?

Reply:
done 9e0b89e